int32   id_index
int8    head
uint64  gps_ms
float64  lat
float64  lon
float64  height
float64  velEast
float64  velNorth
float64  velUp
float64  pitch
float64  roll
float64  yaw
float64  accX
float64  accY
float64  accZ
float64  gyroX
float64  gyroY
float64  gyroZ

int8  navState
int8  gpsState
